### PR TITLE
Apply eslint, fix alignment signup, add smooth wheel zoom, change tag style, change about page sentence and fix mispelling

### PR DESCRIPTION
**Pull request by lhtrung307**
I've rebased some commit that I commit with the wrong email, please help me to check if app.js is right. Sorry for this inconvenience.

### CODE REVIEW

**Comment by xlcrr** [comment]
2 comments here @lhtrung307

File: resources/js/components/Litter/Tags.vue [hunk 1]
```diff
@@ -9,10 +9,11 @@
...
                 <span
                     v-for="tags in Object.entries(category.tags)"
-                    v-html="getTags(tags, category.category)"
-                    class="tag is-large is-info litter-tag"
-                    @click="removeTag(category.category, tags[0])"
-                />
+                    class="tag is-medium is-info litter-tag"
+                >
+                    {{$t(`litter.${category.category}.${tags[0]}`) + ': ' + tags[1]}}
+                    <button class="delete is-small" @click="removeTag(category.category, tags[0])"/>
```
Comment (xlcrr) [inline resources/js/components/Litter/Tags.vue:15]: Keep `v-html` and `@click="removeTag` please

File: resources/js/langs/en/index.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-import signup from './signup.json'
-import tags from './tags.json'
-import teams from './teams.json'
-import nav from './nav.json'
-import upload from './upload.json'
+import signup from './signup.json';
+import tags from './tags.json';
+import teams from './teams.json';
+import nav from './nav.json';
+import upload from './upload.json';
```
Comment (xlcrr) [inline resources/js/langs/en/index.js:14]: Please remove `;` they were left out intentionally

File: resources/js/views/general/Upload.vue [hunk 3]
```diff
@@ -6,12 +6,14 @@
             <h1 class="title is-1 drop-title">{{$t('upload.click-to-upload')}}</h1>
 
             <vue-dropzone :options="options" :useCustomSlot=true id="customdropzone" @vdropzone-error="verror">
-                <i class="fa fa-image upload-icon" aria-hidden="true" />
+                <i class='fa fa-image upload-icon' aria-hidden='true'></i>
```
Comment (xlcrr) [inline resources/js/views/general/Upload.vue:9]: Keep the `"` for classes please

File: resources/js/views/global/Supercluster.vue [hunk 4]
```diff
@@ -17,14 +17,15 @@
...
+import LiveEvents from '../../components/LiveEvents';
 // import GlobalInfo from '../../components/global/GlobalInfo'
-import {CLUSTER_ZOOM_THRESHOLD, MAX_ZOOM, MEDIUM_CLUSTER_SIZE, LARGE_CLUSTER_SIZE, MIN_ZOOM, SHOW_DETAIL_ZOOM, ZOOM_STEP} from '../../constants'
+import {CLUSTER_ZOOM_THRESHOLD, MAX_ZOOM, MEDIUM_CLUSTER_SIZE, LARGE_CLUSTER_SIZE, MIN_ZOOM, SHOW_DETAIL_ZOOM, ZOOM_STEP} from '../../constants';
 
-import L from 'leaflet'
-import moment from 'moment'
+import L from 'leaflet';
+import moment from 'moment';
+import './SmoothWheelZoom.js';
```
Comment (xlcrr) [inline resources/js/views/global/Supercluster.vue:28]: Please remove the `;` they were left out intentionally

**Review by xlcrr — CHANGES_REQUESTED** [review]
Small few changes needed. Cant wait to see smooth zooming!!

File: resources/js/app.js [hunk 5]
```diff
@@ -1,60 +1,65 @@
...
-import axios from 'axios'
-import store from './store'
-import VueRouter from 'vue-router'
-import router from './routes'
-import i18n from './i18n'
-import VueLocalStorage from 'vue-localstorage'
-import VueSweetalert2 from 'vue-sweetalert2'
-import 'sweetalert2/dist/sweetalert2.min.css'
-import VueToastify from 'vue-toastify'
+import Vue from 'vue';
```
[the remark this answers is not in the document]
Comment (lhtrung307) [reply resources/js/app.js:3]: Why you left them out? JS need `;` at the end of the line.

File: resources/js/components/Litter/Tags.vue (hunk 1 above)
Comment (lhtrung307) [reply resources/js/components/Litter/Tags.vue:15]: Move `removeTag` to delete button, no need to remove when to tag.

File: resources/js/langs/en/index.js (hunk 2 above)
Comment (lhtrung307) [reply resources/js/langs/en/index.js:14]: Why you left them out? JS need ; at the end of the line.

File: resources/js/components/Litter/Tags.vue (hunk 1 above)
Comment (lhtrung307) [reply resources/js/components/Litter/Tags.vue:15]: Why do we need to keep `v-html`? We just need text inside the tag.

File: resources/js/views/general/Upload.vue (hunk 3 above)
Comment (lhtrung307) [reply resources/js/views/general/Upload.vue:9]: Updated. Thanks

File: resources/js/app.js [hunk 6]
```diff
@@ -1,60 +1,65 @@
-require('./bootstrap')
+require('./bootstrap',);
```
[the remark this answers is not in the document]
Comment (lhtrung307) [reply resources/js/app.js:1]: Updated. Thanks.

File: resources/js/app.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (xlcrr) [reply resources/js/app.js:3]: It's cleaner without. Doesn't make any difference

File: resources/js/views/Locations/Countries.vue [hunk 7]
```diff
@@ -61,13 +61,13 @@
                     <div class="columns">
                         <div class="column is-half is-offset-3">
                             <div class="columns">
-                                <div class="column is-one-third">
-                                    <h1 class="subtitle is-5" style="color: black; text-align: center;">
```
Comment (xlcrr) [inline resources/js/views/Locations/Countries.vue:65]: Why remove black colour here?

File: resources/js/views/Locations/Countries.vue [hunk 8]
```diff
@@ -81,13 +81,13 @@
                                     </h1>
                                 </div>
 
-                                <div class="column is-one-third">
-                                    <h1 class="subtitle is-5" style="color: black; text-align: center;">
```
Comment (xlcrr) [inline resources/js/views/Locations/Countries.vue:85]: No need to remove black colour here

File: resources/js/views/Locations/Countries.vue [hunk 9]
```diff
@@ -81,13 +81,13 @@
...
+                                <div class="column has-text-centered">
+                                    <h1 class="subtitle is-6 ">
```
Comment (xlcrr) [inline resources/js/views/Locations/Countries.vue:85]: Space here

File: resources/js/components/Litter/Tags.vue [hunk 10]
```diff
@@ -1,21 +1,21 @@
...
+            <li v-for="category in categories" class="admin-item">
                 <!-- Translated Category Title -->
                 <span class="category">{{ getCategory(category.category) }}</span>
 
                 <!-- List of tags in each category -->
                 <span
                     v-for="tags in Object.entries(category.tags)"
-                    v-html="getTags(tags, category.category)"
-                    class="tag is-large is-info litter-tag"
-                    @click="removeTag(category.category, tags[0])"
```
Comment (xlcrr) [inline resources/js/components/Litter/Tags.vue:14]: We need to keep @click="removeTag" here. Also, bring v-html back please. I prefer the readability

File: resources/js/components/Litter/Tags.vue (hunk 1 above)
Comment (xlcrr) [reply resources/js/components/Litter/Tags.vue]: We want to click tag to delete it. It's UX

Comment (xlcrr) [reply resources/js/components/Litter/Tags.vue]: We don't need to keep `-v-html` but I prefer the readability

**Comment by xlcrr** [comment]
1. Remove `;` from js
2. Change back to `v-html`
3. Add back `@click="removeTag"` 
4. Add back `black` colour

and it should be good to merge! Thanks!

File: resources/js/app.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (lhtrung307) [reply resources/js/app.js:3]: It's for consistency. Eslint will add `;` for us at the end of line.

**Review by xlcrr — APPROVED** [review]

**Pull request merged**